src/main/java/org/variantsync/diffdetective/variation/diff/parse/DiffTreeParser.java

```diff
@@ -315,7 +315,7 @@ private void parseLine(
 
         // Is this line a conditional macro?
         // Note: The following line doesn't handle comments and line continuations correctly.
-        var matcher = macroPattern.matcher(line.getLines().get(0));
+        var matcher = macroPattern.matcher(line.toString());
```

Comment:
Is the comment deprecated now? If so, remove it?

Reply:
No this comment is still correct (although in very unlikely use cases). Consider
```c
# /* No does something like this, right? */ \
 ifdef A
// This is legal C and C++
#endif
```